Sources/Sentry/SentryUIViewControllerPerformanceTracker.m

```diff
@@ -37,188 +37,197 @@ - (instancetype)init
 #if SENTRY_HAS_UIKIT
 
 - (void)viewControllerLoadView:(UIViewController *)controller
-              callbackToOrigin:(void (^)(void))callback
+              callbackToOrigin:(void (^)(void))callbackToOrigin
 {
-    [self
-        limitOverride:@"loadView"
-               target:controller
-                block:^{
-                    SentrySpanId *spanId = objc_getAssociatedObject(
-                        controller, &SENTRY_UI_PERFORMANCE_TRACKER_SPAN_ID);
-
-                    // If the user manually call loadView outside the lifecycle
-                    // we don't start a new transaction and override the previous id stored.
-                    if (spanId == nil) {
-                        NSString *name =
-                            [SentryUIViewControllerSanitizer sanitizeViewControllerName:controller];
-                        spanId = [self.tracker
-                            startSpanWithName:name
-                                    operation:SENTRY_VIEWCONTROLLER_RENDERING_OPERATION];
-
-                        // use the target itself to store the spanId to avoid using a global mapper.
-                        objc_setAssociatedObject(controller, &SENTRY_UI_PERFORMANCE_TRACKER_SPAN_ID,
-                            spanId, OBJC_ASSOCIATION_RETAIN_NONATOMIC);
-                    }
-
-                    [self measurePerformance:@"loadView" target:controller blockToMeasure:callback];
-                }];
+    [self limitOverride:@"loadView"
+                  target:controller
+        callbackToOrigin:callbackToOrigin
+                   block:^{
+                       SentrySpanId *spanId = objc_getAssociatedObject(
+                           controller, &SENTRY_UI_PERFORMANCE_TRACKER_SPAN_ID);
+
+                       // If the user manually call loadView outside the lifecycle
+                       // we don't start a new transaction and override the previous id stored.
+                       if (spanId == nil) {
+                           NSString *name = [SentryUIViewControllerSanitizer
+                               sanitizeViewControllerName:controller];
+                           spanId = [self.tracker
+                               startSpanWithName:name
+                                       operation:SENTRY_VIEWCONTROLLER_RENDERING_OPERATION];
+
+                           // use the target itself to store the spanId to avoid using a global
+                           // mapper.
+                           objc_setAssociatedObject(controller,
+                               &SENTRY_UI_PERFORMANCE_TRACKER_SPAN_ID, spanId,
+                               OBJC_ASSOCIATION_RETAIN_NONATOMIC);
+                       }
+
+                       [self measurePerformance:@"loadView"
+                                         target:controller
+                               callbackToOrigin:callbackToOrigin];
+                   }];
```

Comment:
I didn't change anything in this big block and the other blocks below. I just renamed the parameter. Scroll down until my next comment for the actual change in this file.